language-support/hs/bindings/test/DA/Ledger/Tests.hs

```diff
@@ -23,28 +24,31 @@ import Test.Tasty as Tasty (TestName,TestTree,testGroup,withResource,defaultMain
 import Test.Tasty.HUnit as Tasty(assertFailure,assertBool,assertEqual,testCase)
 import qualified Codec.Archive.Zip as Zip
 import qualified DA.Daml.LF.Ast as LF
+import qualified DA.Ledger.Jwt as Jwt
+import qualified Data.Aeson.Types as Aeson
 import qualified Data.ByteString as BS (readFile)
 import qualified Data.ByteString.Lazy as BSL (readFile,toStrict)
 import qualified Data.ByteString.UTF8 as BS (ByteString,fromString)
 import qualified Data.Map as Map
 import qualified Data.Set as Set
-import qualified Data.Text.Lazy as Text(pack,unpack,fromStrict)
+import qualified Data.Text as DT(pack,unpack)
```

Comment:
Importing `Data.Text as DT` and `Data.Text.Lazy as Text` is a bit confusing. I would recommend that you either go with `Data.Text as T` and `Data.Text.Lazy as TL` which we use elsewhere or if you don’t like the short forms `Text` and `Text.Lazy`.